### PR TITLE
Anonymous signin

### PR DESCRIPTION
**Pull request by ksheremet**

### CODE REVIEW

File: flutter/lib/views/decks_list/navigation_drawer.dart [hunk 1]
```diff
@@ -29,23 +34,37 @@ class _NavigationDrawerState extends State<NavigationDrawer> {
...
         child: Column(
       children: <Widget>[
         UserAccountsDrawerHeader(
-          accountName: Text(user.displayName),
-          accountEmail: Text(user.email),
-          currentAccountPicture: CircleAvatar(
-            backgroundImage: NetworkImage(user.photoUrl),
-          ),
+          accountName: Text(accountName),
+          accountEmail: Text(user.email ?? ''),
```
Comment (dotdoom) [inline flutter/lib/views/decks_list/navigation_drawer.dart]: `accountEmail: user.email == null ? null : Text(user.email)`

File: flutter/lib/views/decks_list/navigation_drawer.dart [hunk 2]
```diff
@@ -29,23 +34,37 @@ class _NavigationDrawerState extends State<NavigationDrawer> {
...
+          currentAccountPicture: user.photoUrl == null
```
Comment (dotdoom) [inline flutter/lib/views/decks_list/navigation_drawer.dart]: How about

`CircleAvatar(backgroundImage: user.photoUrl == null ? AssetImage(...) : NetworkImage(...)` ?

**Comment by dotdoom** [comment]
`Small fixes` :cat2:

**Comment by ksheremet** [comment]
screenshot_1538932392

**Pull request merged**